### ISSUE TITLE
Make a separate "Users Guide" README for people that want to just DL & run it

### ISSUE THREAD

**Issue by orta** [realm/jazzy#114]

**Comment by mattgabor** [comment]
I am confused about how to view the documentation. I ran jazzy --swift-version 2.1 in my project directory and it created a folder called docs, but when I open index.html, it looks like this:

<img width="675" alt="screen shot 2015-10-28 at 5 50 42 pm" src="https://cloud.githubusercontent.com/assets/5590876/10807393/6ee0fa64-7d9c-11e5-8894-7e8fb8cf387c.png">


I have my comments in the Apple Markdown format, but nothing shows up. Is there a separate README that discusses installation and formatting I am missing?

**Comment by jpsim** [comment]
No, I never ended up taking the time to write something like that. My guess is that you don't have any declarations in your module with a `public` ACL, or that the scheme detected by jazzy isn't what you want to document. You can address either of these issues by passing `--min-acl` and `--module` to jazzy respectively.

**Comment by mattgabor** [comment]
Thanks @jpsim, I am still having this issue. I run  `jazzy --swift-version 2.1 --min-acl --module TheGame`
and jazzy runs properly, but my docs are still empty. Just to confirm, is this the format I should be using to document my code? 
```
   /**
    Searches the file system for a given directory
    - author: Matt Gabor
    - parameter atPath: The path you want to retrieve files from
    - returns: An array of the path URLs located in atPath
    */
```

**Comment by jpsim** [comment]
@mattgabor that looks like an appropriate format. Please share *full* steps that I can follow to reproduce the issue you're having, otherwise it'll continue to be quite difficult for me to help you troubleshoot.

**Comment by mattgabor** [comment]
I had to specify a min-ace with --min-acl internal, not just --min-acl

**Comment by jpsim** [comment]
Ok, thanks for letting us know! That should be made clearer with #337.

**Comment by istx25** [comment]
Resolved with #337. Closing.
